pkg/pipelines/bootstrap.go

```diff
@@ -12,9 +12,19 @@ import (
 	"sigs.k8s.io/yaml"
 )
 
+var DefaultTokenFilename string = "~/Downloads/token"
+var DefaultQuayIOAuthFilename string = "~/Downloads/<username>-auth.json"
+
```

Comment:
I don't know if it makes sure to provide default values if the defaults are guaranteed not going to work.   If they are just a doc tool then they are fine.